addon/components/select-input.js

```diff
@@ -50,20 +50,23 @@ export default AbstractInput.extend({
 
   didReceiveAttrs ({oldAttrs, newAttrs}) {
     this._super(...arguments)
+
     const modelDef = this.get('bunsenModel')
     if (!modelDef) {
       return
     }
-    const dbStore = this.get('dbStore')
-    const value = this.get('bunsenStore.formValue')
-    const bunsenId = this.get('bunsenId')
-
-    if (this.hasQueryChanged(oldAttrs, newAttrs, modelDef.query) &&
-      utils.hasValidQueryValues(value, modelDef.query, bunsenId)) {
-      listUtils.getOptions(value, modelDef, bunsenId, dbStore).then((opts) => {
-        this.set('options', opts)
-      })
+    if (this.hasQueryChanged(oldAttrs, newAttrs, modelDef.query)) {
+           // setting required variables once above condition is true
```

Comment:
Are you referring to to the extra space @ line 59 
"// setting required variables once above condition is true"